pandas/io/feather_format.py

```diff
@@ -96,17 +103,23 @@ def read_feather(path, nthreads=1):
         Number of CPU threads to use when reading to pandas.DataFrame
 
        .. versionadded 0.21.0
+       .. deprecated 0.24.0
+    use_threads: bool, default True
+        Whether to parallelize reading using multiple threads
+
+       .. versionadded 0.24.0
 
     Returns
     -------
     type of object stored in file
 
     """
 
-    feather = _try_import()
+    feather, pyarrow = _try_import()
     path = _stringify_path(path)
 
-    if LooseVersion(feather.__version__) < LooseVersion('0.4.0'):
+    if LooseVersion(feather.__version__) < LooseVersion('0.4.0') or \
```

Comment:
Style nit: start the line with parentheses instead of using a backslash to continue the line.

Reply:
Do you have a suggestion for the alternative? It does not line up as nicely (since it would require an additional indent of the second line). Making it in my opinion harder to read.

Reply:
```
if (LooseVersion(feather.__version__) < LooseVersion('0.4.0') or
        LooseVersion(pyarrow.__version__) < LooseVersion('0.11.0')):
    return feather.read_dataframe(path)
```